### ISSUE TITLE
Travis tests are not running

### ISSUE THREAD

**Issue by rohityadavcloud** [apache/cloudstack#6097]
Check with Travis and ensure following:
- Smoketests should run
- UI tests should run
- Lint checks if any should run

... and that the build MUST fail if any of the above fail.

For example, right now smoketests aren't running - https://app.travis-ci.com/github/apache/cloudstack/builds/247736668

Pl check and fix this for both 4.16 and 4.17/main as applicable.

##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
CI/CD
~~~

**Comment by nvazquez** [comment]
Hi @rohityadavcloud I see tests running now, a lot of them are still queued

**Comment by rohityadavcloud** [comment]
No @nvazquez cc @weizhouapache @Pearl1594 @davidjumani pl look at individual jobs, for example - https://app.travis-ci.com/github/apache/cloudstack/builds/247795805

They're all running in under 20mins but they are not really running Marvin smoketests, from one of those "passing" jobs we see:

```
nosetests: error: no such option: --with-marvin
real	0m0.173s
user	0m0.126s
sys	0m0.020s
```

**Comment by nvazquez** [comment]
Closing it since its been fixed on #6113
